2020/day1/solution.py

```diff
@@ -1,15 +1,14 @@
 file = open('input.txt')
-numbers = map(int, file.readlines())
-number_map = dict(map(lambda x : (x, 1), numbers))
-for num in number_map:
-    if 2020 - num in number_map:
+numbers = set(map(int, file.readlines()))
```

Comment:
```python
numbers = {int(line) for line in file.readlines()}
```

Reply:
ooh, I can just do a comprehension to build here, good point. thanks